mozci/push.py

```diff
@@ -1132,10 +1132,15 @@ def __make_group_summary_objects(pushes):
     """
     # Obtain all the task.id values contained in the pushes. It will be used in
     # the `where` query against ActiveData.
-    task_ids = [task.id for push in pushes for task in push.tasks]
-
-    # Extract the results of the above query.
-    results = run_query("group_durations", Namespace(task_id=task_ids))["data"]
+    # task_ids = [task.id for push in pushes for task in push.tasks]
+    results = []
+    revs = [p.rev for p in pushes]
+    for rev in revs:
+        try:
+            result = run_query("group_durations", Namespace(push_ids=rev))["data"]
```

Comment:
Instead of making one query per rev, could we do a single query with all the revs?